server/konnectors/uber.js

```diff
@@ -86,7 +86,7 @@ function logIn(requiredFields, bills, data, next) {
       if (res.statuCode >= 400) {
         log.error('Login failed');
         const err = `Status code: ${res.statusCode}`;
-        log.error(err);
+        log.error('request error');
         return next(err);
```

Comment:
Is there any reason you return the error and log 'request error' instead of the contrary ? 
Maybe 1 line of log would be enough, too ?

```js
log.error(`Login failed due to request error (status code: ${res.statusCode)`);
```

Same thing should be done there too: https://github.com/cozy-labs/konnectors/blob/eb33c3eeacb6c9f156c509c5096fcda26d291608/server/konnectors/uber.js#L62